src/build-vrunner2.sh

```
@@ -0,0 +1,113 @@
+set -eo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+if [ -z "${CI:-}" ]; then
+    echo "The script is not running in CI"
+    source "${SCRIPT_DIR}/../scripts/load_env.sh"
+else
+    echo "The script is running in CI"
+fi
+
+# Логи и ассерт-хелперы
+source "${SCRIPT_DIR}/../tools/assert.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../tools/assert.sh was not specified as input (see shellcheck -x).

(SC1091)